### PR TITLE
Seata document

### LINKED ISSUE #5559: add more document for seata
## Feature Request

I saw there are some more detailed document for seata, so  I would like to add these to shardingsphere transaction document

### PR DESCRIPTION
**Pull request by taojintianxia**
Fixes #5559.

- add a little bit seata document

### CODE REVIEW

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md [hunk 1]
```diff
@@ -13,4 +13,14 @@ weight = 3
 `Seata AT`事务模型包含TM(事务管理器)，RM(资源管理器)，TC(事务协调器)。其中TC是一个独立的服务需要单独部署，TM和RM以jar包的方式同业务应用部署在一起，它们同TC建立长连接，在整个事务生命周期内，保持RPC通信。
 其中全局事务的发起方作为TM，全局事务的参与者作为RM ; TM负责全局事务的begin和commit/rollback，RM负责分支事务的执行结果上报，并且通过TC的协调进行commit/rollback。
 
+A typical lifecycle of Seata managed distributed transaction:
```
[the remark this answers is not in the document]
Comment (taojintianxia) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md]: ops ！sorry

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md [hunk 2]
```diff
@@ -13,4 +13,12 @@ weight = 3
 `Seata AT`事务模型包含TM(事务管理器)，RM(资源管理器)，TC(事务协调器)。其中TC是一个独立的服务需要单独部署，TM和RM以jar包的方式同业务应用部署在一起，它们同TC建立长连接，在整个事务生命周期内，保持RPC通信。
 其中全局事务的发起方作为TM，全局事务的参与者作为RM ; TM负责全局事务的begin和commit/rollback，RM负责分支事务的执行结果上报，并且通过TC的协调进行commit/rollback。
 
+Seata管理的分布式事务的典型生命周期：
+
+1. TM要求TC开始一个全新的全局事务。 TC生成一个代表该全局事务的XID。
+2. XID 通过微服务贯穿整个调用链。
```
Comment (kimmking) [inline docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:19]: XID 贯穿于微服务的整个调用链。

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md [hunk 3]
```diff
@@ -13,4 +13,12 @@ weight = 3
 `Seata AT`事务模型包含TM(事务管理器)，RM(资源管理器)，TC(事务协调器)。其中TC是一个独立的服务需要单独部署，TM和RM以jar包的方式同业务应用部署在一起，它们同TC建立长连接，在整个事务生命周期内，保持RPC通信。
 其中全局事务的发起方作为TM，全局事务的参与者作为RM ; TM负责全局事务的begin和commit/rollback，RM负责分支事务的执行结果上报，并且通过TC的协调进行commit/rollback。
 
+Seata管理的分布式事务的典型生命周期：
+
+1. TM要求TC开始一个全新的全局事务。 TC生成一个代表该全局事务的XID。
```
Comment (kimmking) [inline docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:18]: one more whitespace.

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md [hunk 4]
```diff
@@ -13,4 +13,12 @@ weight = 3
...
+2. XID 通过微服务贯穿整个调用链。
+3. 作为该XID对应到的TC下的全局事务的一部分，RM注册了本地事务。
+4. TM要求TC提交或回退XID对应的全局事务。
```
Comment (kimmking) [inline docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:21]: 回退 -> 回滚

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md (hunk 2 above)
Comment (taojintianxia) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:19]: at very begining , I translated as same as yours . however ... enn.

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md (hunk 3 above)
Comment (taojintianxia) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:18]: what you mean one more whitespace . add after the colon ？

Comment (kimmking) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:18]: between `。 TC`

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md (hunk 4 above)
Comment (kimmking) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:21]: `TM要求TC提交或回退XID对应的全局事务。`

回退 -> 回滚

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md (hunk 3 above)
Comment (taojintianxia) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md]: damn .... how do you find that. :sweat_smile:

File: docs/document/content/features/transaction/concept/base-transaction-seata.cn.md [hunk 5]
```diff
@@ -13,4 +13,12 @@ weight = 3
 `Seata AT`事务模型包含TM(事务管理器)，RM(资源管理器)，TC(事务协调器)。其中TC是一个独立的服务需要单独部署，TM和RM以jar包的方式同业务应用部署在一起，它们同TC建立长连接，在整个事务生命周期内，保持RPC通信。
 其中全局事务的发起方作为TM，全局事务的参与者作为RM ; TM负责全局事务的begin和commit/rollback，RM负责分支事务的执行结果上报，并且通过TC的协调进行commit/rollback。
 
+Seata管理的分布式事务的典型生命周期：
+
+1. TM要求TC开始一个全新的全局事务。TC生成一个代表该全局事务的XID。
+2. XID 贯穿于微服务的整个调用链。
```
Comment (cherrylzhao) [inline docs/document/content/features/transaction/concept/base-transaction-seata.cn.md:19]: duplicate whitespace exist after XID ?

Comment (taojintianxia) [reply docs/document/content/features/transaction/concept/base-transaction-seata.cn.md]: thx, fixed

**Pull request merged**